packages/workbox-webpack-plugin/test/node/e2etest.js

```diff
@@ -0,0 +1,84 @@
+/* global workbox, expect, describe */
+const assert = require('chai').assert;
+const WorkboxBuild = require('../../../workbox-build/src');
+const proxyquire = require('proxyquire');
+const webpack = require('webpack');
+const fs = require('fs');
+const path = require('path');
+
+let WorkboxWebpackPlugin;
+
+function runWebpack(webpackConfig) {
+  return new Promise((resolve, reject) => {
+    let compiler = webpack(webpackConfig);
+    compiler.run((err, stats)=>{
+      if (err) reject(err);
+      else {
+        // Timeout for plugins that work on `after-emit` event of webpack
+        setTimeout(()=>{
+          resolve(stats);
+        }, 20);
+      }
+    });
+  });
+}
+
+describe('Tests for webpack plugin', function() {
+  this.timeout(55000);
+  beforeEach(()=>{
+    // do a proxy require
+    WorkboxWebpackPlugin = proxyquire('../../', {
+      'workbox-build': WorkboxBuild,
+    });
+    new WorkboxWebpackPlugin();
+  });
+
+  it('should generate sw, when `swSrc` is not present', () => {
+    this.timeout(35000);
```

Comment:
Why does this test have a timeout of 35 seconds, after you set a timeout of 55 seconds at the beginning of the `describe()` block? And what is the significance of those values—is there any reason why running a Webpack build would legitimately take 55 seconds to complete?